OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -35,6 +35,12 @@ class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientation
 		[Desc("Offset from the actor the missile spawns on.")]
 		public readonly WVec SpawnOffset = WVec.Zero;
 
+		[Desc("Altitude offset from the target position at which the warhead should detonate.")]
+		public readonly WDist PreDetonationAltitude = WDist.Zero;
```

Comment:
I preferred the idea of having separate `AirburstAltitude` and `PreDetonationAltitude`, but if you want to have them combined then this should be kept as `AirbustAltitude` and `RemoveMissileOnAirburst`.

The whole motivation for the "PreDetonation" name was that it would never detonate/remove the missile.

Reply:
On second thought, the better option is to just remove the "Pre". `DetonationAltitude` and `RemoveMissileOnDetonation` sound much better than Airburst, which we only considered because of the RA2 history.